installer/README.md

```diff
@@ -7,14 +7,22 @@
   * The resulting installer will be built to `PowerToysSetup\bin\Release\PowerToysSetup.msi`.
 
 ## Building and installing self-signed PowerToys MSIX package
-For the first-time installation, you should generate a self-signed certificate and add it to the [TRCA store](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/trusted-root-certification-authorities-certificate-store). That could be done by simply running ` 
-generate_self_sign_cert.ps1` from a powershell admin. After that:
+For the first-time installation, you'll need to generate a self-signed certificate.  The script below will generate and add a cert to your [TRCA store](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/trusted-root-certification-authorities-certificate-store). 
+1. Open PowerShell as an Admin
+2. Navigate to your repo's `installer\MSIX`
+3. Run `.\generate_self_sign_cert.ps1`
+4. Add your Windows SDK to your Path variable based on your most recent SDK.  Example:  `%ProgramFiles(x86)%\Windows Kits\10\bin\10.0.18362.0\x86`
```

Comment:
When installing VS 2019, there is already a PowerShell link that has the path set
![image](https://user-images.githubusercontent.com/3206696/72414702-233a2000-3773-11ea-9609-f06627b4bdcb.png)

Reply:
I would suggest to add it as an option to 4.:
```
4. Run the `Developer PowerShell for VS` or add your Windows SDK to your Path variable based on your most recent SDK.  Example:  `%ProgramFiles(x86)%\Windows Kits\10\bin\10.0.18362.0\x86`
```
This steps is needed for the following Build command, shouldn't we move it there?

Reply:
Another option is to run the `x64 Native Tools Command Prompt` and then run PowerShell from there, without having to manually add the SDK path to the PATH env variable.

Reply:
I'm now considering that we shouldn't suggest to add the SDK path to the PATH env variable since this would break the way the VS command prompts work.